crates/sui-tool/src/lib.rs

```diff
@@ -726,32 +739,34 @@ fn start_summary_sync(
             }
         });
 
-        let sync_range = s_start..last_checkpoint + 1;
-        archive_reader
-            .read_summaries(
-                state_sync_store.clone(),
-                sync_range.clone(),
-                sync_checkpoint_counter,
-                // rather than blocking on verify, sync all summaries first, then verify later
-                false,
-            )
-            .await?;
+        if with_skiplist {
+            archive_reader
+                .read_summaries_with_skiplist(
+                    state_sync_store.clone(),
+                    end_of_epoch_checkpoint_seq_nums.clone(),
+                    sync_checkpoint_counter,
+                )
+                .await?;
+        } else {
+            archive_reader
+                .read_summaries_with_range(
+                    state_sync_store.clone(),
+                    s_start..last_checkpoint + 1,
+                    sync_checkpoint_counter,
+                    // rather than blocking on verify, sync all summaries first, then verify later
+                    false,
```

Comment:
Is there still a case where true is passed? Trying to understand if you still need this argument

Reply:
No this is the only callsite, but I had thought to leave the option in `read_summaries_with_range` with verification for the sake of generality. I also get concerned that someone may use the fn in the future without realizing that it is leaving verification to the caller.

I'll remove the verify path in here, but be more explicit in the naming